backend/icpy/api/rest_api.py

```
@@ -579,51 +653,81 @@ async def create_file(request: FileOperationRequest):
                         raise Exception("Failed to create directory")
                 else:
                     # Create file (existing logic)
-                    await self.filesystem_service.write_file(
+                    await fs.write_file(
                         file_path=request.path,
                         content=request.content or "",
                         encoding=request.encoding,
                         create_dirs=request.create_dirs
                     )
                     return SuccessResponse(message="File created successfully")
             except Exception as e:
```

Ruff (0.13.2)

Do not catch blind exception: `Exception`

(BLE001)